src/mumble/PulseAudio.cpp

```diff
@@ -566,15 +566,23 @@ void PulseAudioSystem::write_callback(pa_stream *s, size_t bytes, void *userdata
 	PulseAudioSystem *pas = reinterpret_cast< PulseAudioSystem * >(userdata);
 	Q_ASSERT(s == pas->pasOutput);
 
+	const auto &pa = pas->m_pulseAudio;
+
 	AudioOutputPtr ao     = Global::get().ao;
 	PulseAudioOutput *pao = dynamic_cast< PulseAudioOutput * >(ao.get());
 
 	if (!pao) {
+		// The AudioSystem pointer is invalid, probably because of a AudioSystem restart.
+		// However, PA expects this callback to return exactly the provided number of
+		// bytes. Failing to do so will cause PA to never call this again,
+		// effectively removing the audio output completely until Mumble is restarted.
+		// See: https://github.com/mumble-voip/mumble/issues/4883
+		// See: https://gitlab.freedesktop.org/pulseaudio/pulseaudio/-/issues/1132
+		unsigned char buffer[bytes] = {};
```

Comment:
This is not standard C++, but a compiler extension (variable-size arrays). This will have to be changed in #5960

(Just take this as a note though - I see the rest of the file uses this anyway, so let's not bother at this point yet)